.moai/reports/t232/tmplgrep.py

```
@@ -0,0 +1,11 @@
+import json
+import os
+import subprocess
+d = json.load(open('.moai/reports/t232/analysis-repro.json'))
+base = 'internal/template/templates'
+ok = 0
+for e in d:
+    t = os.path.join(base, e['file'])
+    if os.path.exists(t) and subprocess.run(['grep', '-F', '-q', '--', e['clause'], t]).returncode == 0:
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)